apis/scheduling/v1alpha1/doc.go

```diff
@@ -14,8 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// +k8s:deepcopy-gen=package
+// +kubebuilder:object:generate=true
 // +groupName=scheduling.sigs.k8s.io
```

Comment:
Is `groupName` needed for kubebuilder? if not, let's remove it.

Reply:
Yes, it's needed